### DISCUSSION TITLE
Customize related attributes?

### DISCUSSION THREAD

**Discussion by jansgescheit** [owen-it/laravel-auditing#819] [Q&A] [answered]
I am currently evaluating this great package. One reason why I am looking at it more closely is the possibility to monitor changes to the related attributes. However, I now have a question about this. 

When I use `auditSync(...)` the complete model is stored serialised in the old/new values. Is there a way to limit this with this package? 
I would rather see the pivot than the model behind it, e.g.

``` json
[{
    article_id: 1, 
    category_id: 1, 
    sort: 2
}, {
    article_id: 1, 
    category_id: 2, 
    sort: 1
}]
```

or just the fields that would be relevant like `auditSync($relationName, $ids, array $columns, $detaching = true)`

I can of course write my own `auditSync` method, but I'm not sure how this would work in a possible transition. Maybe someone here has an idea.

**Comment by angeljqv** [comment, marked as answer]
here https://laravel-auditing.com/guide/audit-transformation.html
You can collect only your relevant data as you wish there
